src/clj/game/core/actions.clj

```diff
@@ -15,8 +15,8 @@
     (case (:type card)
       ("Event" "Operation") (play-instant state side card {:extra-cost [:click 1]})
       ("Hardware" "Resource" "Program") (runner-install state side (make-eid state) card {:extra-cost [:click 1]})
-      ("ICE" "Upgrade" "Asset" "Agenda") (corp-install state side card server {:extra-cost [:click 1]}))
-    (trigger-event state side :play card)))
+      ("ICE" "Upgrade" "Asset" "Agenda") (corp-install state side card server {:extra-cost [:click 1]})))
+  (trigger-event state side :play card))
```

Comment:
No intent.  Failed cleanup of my earlier issue with "do" on this function. ie, It was not needed.